SCF-Session-5/lcmhcf/LcmHcf.java

```diff
@@ -0,0 +1,44 @@
+package lcmhcf;
+
+/**
+ * This class contains methods to find LCM and HCF of two numbers using recursion.
+ * @author Ishita Mathur
+ *
+ */
+public class LcmHcf 
+{
+	static int i = 1;
```

Comment:
Why `static int i = 1` is declared at class level?

Reply:
It was discussed in meeting